di_website/templates/includes/partials/multi-authors.html

```diff
@@ -2,24 +2,18 @@
 
 <div class="m-authors">
     <h3 class="m-authors__title">
-      {% if first_author and authors|length > 0 %}
-        Authors
-      {% elif authors|length > 1 %}
-        Authors
-      {% else %}
-        Author
-      {% endif %}
```

Comment:
You really should be careful with editing reusable templates ... such changes have far-reaching effects ... I'm sure this original code was there for a reason. For this entire file ... come on... with great power!!

Reply:
Restored deleted html and added conditional for multi author blogs